ocs_ci/deployment/baremetal.py

```diff
@@ -1251,7 +1251,7 @@ def destroy_cluster(self, log_level="DEBUG"):
 
 
 @retry(exceptions.CommandFailed, tries=10, delay=30, backoff=1)
-def clean_disk(worker):
+def clean_disk(worker, namespace=constants.BM_DEBUG_NODE_NS):
```

Comment:
we would need to add new argument to docstring; 
but I don't think that we would need this additional parameter at all 
you don't use it in this PR https://github.com/red-hat-storage/ocs-ci/pull/9144/files#:~:text=clean_disk(node_obj)